catalog/docs/src/main/resources/ExtContents.adoc

```diff
@@ -3295,6 +3295,46 @@ application/vnd.oasis.opendocument.text
 ====== Implementation Details
 This input transformer maps the metadata common across all mime types to applicable metacard attributes in the default MetacardType.
 
+===== Video Input Transformer
+The video input transformer is responsible for creating Catalog metacards from certain video file types. Currently, it is responsible for handling MPEG-2 transport streams as well as MPEG-4, AVI, MOV, and WMV videos. This input transformer uses Apache Tika to extract basic metadata from the video files and applies more sophisticated methods to extract more meaningful metadata from these types of video.
+
+====== Installing and Uninstalling
+Install the `catalog-transformer-video` feature using the Web Console (http://localhost:8181/system/console) or System Console. This feature is installed by default.
```

Comment:
Do not call out using the /system/console and do not call it the web console. Use "Admin Console" and https://localhost:8993/admin. Provide steps to how the user would get to this feature from the Admin Console.